pkg/kubelet/kubeletconfig/status/status.go

```diff
@@ -142,6 +145,8 @@ func (s *nodeConfigStatus) Sync(client clientset.Interface, nodeName string) {
 		return
```

Comment:
I realize this isn't part of this PR, but is there a reason you chose to use a periodic, jittered check rather than just a long-running loop?
`for _ := range s.syncCh {`
`    utillog.Infof("updating Node.Status.Config")`
` ...`
`}`